Makefile

```diff
@@ -60,6 +62,18 @@ dev-prerequisites:
 unit-tests: dev-prerequisites
 	go test -v -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=./... ./...
 
+.PHONY: coverage-dump
+coverage-dump: unit-tests
+	go tool cover -func=coverage.txt
+
+.PHONY: coverage-report
+coverage-report: unit-tests
+	go tool cover -o coverage.html -html coverage.txt
+
+.PHONY: coverage-dump
+coverage-dump: unit-tests
```

Comment:
It looks like this stanza is a duplicate of the one in line 65. We probably only need one "coverage-dump" target or am I missing something? (copy/paste issue maybe?)